libqtile/widget/battery.py

```diff
@@ -61,9 +61,10 @@ class _Battery(base._TextBox):
     """Base battery class"""
 
     filenames = {}
+    # ACPI name of a battery, usually BAT0
+    battery_name = 'BAT0'
 
     defaults = [
-        ('battery_name', 'BAT0', 'ACPI name of a battery, usually BAT0'),
```

Comment:
Can we just s/BAT0/get_battery_name()/ here? That way we will still get all the docs autogenerated for this from config.

Reply:
maybe, i will try it

Reply:
_get_battery_name() replaced BAT0